### PR TITLE
[WIP] Pull-in some changes from upstream

### PR DESCRIPTION
**Pull request by skateman**
I'm not sure if I'm doing it the right way, but there is a [bug](https://github.com/ManageIQ/manageiq-ui-classic/issues/7015) in our current version and I'm hoping that this would fix it.

### CODE REVIEW

**Comment by skateman** [comment]
@himdel I'm not entirely sure if we need any other change to make this working, or you have some kind of magic stuff that builds our NPM package from this. I tried to find it, but NPM is down :confused: 

Anyway, would it be possible to create an appliance from this somehow? I guess I need to change things in the ui-classic and ask Satoe, but I am suspecting that there's something more to do.

**Comment by skateman** [comment]
Maybe this repo will be unnecessary, I found out that they made an [npm package](https://www.npmjs.com/package/@spice-project/spice-html5)

File: spice.html [hunk 1]
```diff
@@ -28,38 +28,25 @@
...
 
+        <!-- ES2015/ES6 modules polyfill -->
+        <script type="module">
+            window._spice_has_module_support = true;
+        </script>
         <script>
+            window.addEventListener("load", function() {
+                if (window._spice_has_module_support) return;
+                var loader = document.createElement("script");
+                loader.src = "thirdparty/browser-es-module-loader/dist/" +
```
Comment (himdel) [inline spice.html:41]: this won't work in manageiq.. but it's probably ok, because module imports should be supported in all supported browsers

**Comment by himdel** [comment]
Switching to the other package then?

But, this is not the right package anyway, we're using https://github.com/himdel/spice-html5-bower

(List of our changes: https://github.com/himdel/spice-html5-bower#local-changes)

**Comment by skateman** [comment]
RHV no longer supports SPICE-HTML5, and actually any web consoles, so it's not really necessary. I don't even know how we could test it...

**Pull request closed without merging**